src/core/services/google-maps-types.ts

```diff
@@ -125,16 +125,27 @@ export interface MapOptions {
   maxZoom?: number;
   disableDoubleClickZoom?: boolean;
   disableDefaultUI?: boolean;
+  scrollwheel?: boolean;
   backgroundColor?: string;
   draggable?: boolean;
   draggableCursor?: string;
   draggingCursor?: string;
   keyboardShortcuts?: boolean;
-  zoomControl?: boolean;
   styles?: MapTypeStyle[];
+  zoomControl?: boolean;
+  zoomControlOptions?: PositionOption;
   streetViewControl?: boolean;
+  streetViewControlOptions?: PositionOption;
   scaleControl?: boolean;
+  scaleControlOptions?: PositionOption;
   mapTypeControl?: boolean;
+  mapTypeControlOptions?: PositionOption;
+  panControl?: boolean;
+  panControlOptions?: PositionOption;
```

Comment:
These new *Options are not directly PositionOptions. The all have a position attribute with a type of ControlPosition (but should also be the explicit types like in the google maps api)